manifests/s/sebescudie/GammaLauncher/5.2.0-preview02/sebescudie.GammaLauncher.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: sebescudie.GammaLauncher
+PackageVersion: 5.2.0-preview02
+PackageLocale: en-US
+Publisher: sebescudie
+PublisherUrl: https://github.com/sebescudie/GammaLauncher
+PublisherSupportUrl: https://github.com/sebescudie/GammaLauncher/issues
+Author: sebescudie
+PackageName: GammaLauncher
+PackageUrl: https://github.com/sebescudie/GammaLauncher
+License: GPL-3.0 License
+LicenseUrl: https://github.com/sebescudie/GammaLauncher/blob/main/LICENSE.md
+Copyright: https://github.com/sebescudie/GammaLauncher
+CopyrightUrl: https://github.com/sebescudie/GammaLauncher
+ShortDescription: GammaLauncher Setup
+Description: A standalone launcher for vvvv gamma
+Moniker: GammaLauncher
+Tags:
+- vvvv gamma launcher
+ReleaseNotes: 5.2.0 preview 02
+ReleaseNotesUrl: https://github.com/sebescudie/GammaLauncher/releases/tag/5.2.0-preview02
+InstallationNotes: Have a delightful patching day
+Icons:
+- IconUrl: https://raw.githubusercontent.com/sebescudie/GammaLauncher/main/ico.png
+  IconFileType: png
+  IconResolution: custom
+  IconTheme: default

```

Comment:
```suggestion
```
I don't know that we're accepting icons yet.

Reply:
Ok, thanks for the feedback.